src/main/java/org/jitsi/impl/protocol/xmpp/XmppProtocolActivator.java

```diff
@@ -44,12 +44,11 @@
 
     static BundleContext bundleContext;
 
-    @Override
-    public void start(BundleContext bundleContext)
-        throws Exception
+    /**
+     * Registers PacketExtension providers used by Jicofo
+     */
+    static public void registerXMppExtensions()
```

Comment:
Was the capitalization of XMpp in registerXMppExtensions intentional? I would've expected XMPP or Xmpp.